src/FloatingWindows/FLWIntegration.cpp

```diff
@@ -201,10 +200,9 @@ void LuaSetOnDrawCallback(sol::light<FloatingWindow> wnd, CallbackProvider const
 
         auto on_draw = on_draw_provider();
         if (!on_draw) {
-            std::ostringstream oss_window_id;
-            oss_window_id << "FlyWithLua Info: This XPLMWindowID " << window << " is for LuaCreateFloatingWindow";
-            flywithlua::logMsg(logToAll, oss_window_id.str());
             flywithlua::panic("FlyWithLua Error: invalid or nil window builder passed to float_wnd_set_ondraw");
+            // Need to find a way for this to work when we are using sol.
+            // FindAndQuarantine (L);
```

Comment:
Need to find how to get to the stack info when we are using sol to finish up finding all the errors.